src/components/ModalCloseButton.js

```diff
@@ -3,10 +3,10 @@ import { TouchableOpacity, StyleSheet } from 'react-native'
 import { MaterialIcons } from '@expo/vector-icons'; 
 
 
-const ModalCloseButton = ({toCloseModal}) => {
+const ModalCloseButton = ({onPress}) => {
   return (
     <TouchableOpacity
-      onPress={toCloseModal}
+      onPress={onPress}
     >
```

Comment:
Change the names of simple callback functions to 'onPress', to make things more simple.